src/ActionModal/utils/ActionManager.js

```diff
@@ -192,7 +192,7 @@ export default class ActionManager {
     if (result.success) {
       return { hash: result.hash }
     } else {
-      throw Error("Broadcast was not successfull")
+      throw Error("Broadcast was not successfull: " + result.error)
```

Comment:
successful only has one 'l'

also, i hope one day all of our error messages can be human and actionable as opposed to just helping us debug problems. 😕